lib/Minz/ExtensionManager.php

```diff
@@ -102,6 +102,10 @@ final class Minz_ExtensionManager {
 			'list' => [],
 			'signature' => 'PassArguments',
 		],
+		'view_modes' => [	// function($viewModes = array) -> array | null
```

Comment:
Please put the same in the documentation
* https://github.com/tryallthethings/freshrss/blob/80fed12a3493da3a64dfb3de61bee4614ea14db9/docs/en/developers/03_Backend/05_Extensions.md
* https://github.com/tryallthethings/freshrss/blob/80fed12a3493da3a64dfb3de61bee4614ea14db9/docs/fr/developers/03_Backend/05_Extensions.md